python/docs/conf.py

```diff
@@ -21,8 +21,12 @@
 copyright = '2020, Microsoft'
 author = 'Microsoft'
 
+with open(os.path.join('..', 'interpret_community', 'version.py')) as f:
```

Comment:
Should this be available through setup.py?

Reply:
hmm no, but actually it is available in the __init__ file if the package is installed

Reply:
just checked the readthedocs build, although it installs the requirements it seems it never actually installs interpret-community in the environment for doc generation, so using ```__init__``` file won't work and the way it is done in this PR seems to be the only way